baseband/vlbi_base/encoding.py

```diff
@@ -11,19 +11,25 @@
 
 # The high mag value for 2-bit reconstruction.
 OPTIMAL_2BIT_HIGH = 3.3359
-r"""Optimal high value for a 2-bit digitizer for which the low value is 1.
+r"""Optimal high value for a 2-bit digitizer for which the low value is
++/- 1.
```

Comment:
Why the added line break here? The actual text on this line is less than 72 characters (I'm not sure I see the point of the +/-, then the high value should also be +/-...)